pypesto/visualize/model_fit.py

```diff
@@ -0,0 +1,53 @@
+import petab
+from amici.petab_simulate import simulate_petab
+import amici.petab_import as petab_import
+from petab.visualize import plot_with_vis_spec
+from typing import Union, Sequence
+from ..result import Result
+from amici.petab_objective import rdatas_to_simulation_df
+
+
+def visualize_optimized_model_fit(petab_problem: petab.Problem,
+                                  result: Union[Result, Sequence[Result]],
+                                  **kwargs
+                                  ):
+    """
+    Function calls the petab.visualize file of the petab_problem and
+    visualizes the fit of the optimized parameter. Possible additional
+    argument is `subplot_dir` to specify the directory each subplot is
```

Comment:
"further keyword arguments are delegated to ... see there for further information" but yes makes sense to document commonly used arguments like subplot_dir here.